pyproject.toml

```diff
@@ -24,14 +24,22 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.8, <3.13"
```

Comment:
python 3.13?

Reply:
was required for `pyinstaller` but not relevant anymore; will remove